samples/bootloader/src/main.c

```diff
@@ -178,6 +201,14 @@ void main(void)
 		return;
 	}
 
+#ifdef CONFIG_SB_DISABLE_FLASH_PATCH
+	if ((NRF_UICR->DEBUGCTRL & UICR_DEBUGCTRL_CPUFPBEN_Msk) != 0x0) {
```

Comment:
Some comment here on what this does and why would be good.

Reply:
Added a comment (in the new function)